proxy/proxy.go

```diff
@@ -909,9 +916,7 @@ func (p *Proxy) do(ctx *context) error {
 	} else {
 		done, allow := p.checkBreaker(ctx)
 		if !allow {
-			if span := ot.SpanFromContext(ctx.Request().Context()); span != nil {
-				span.LogKV(`circuit_breaker`, `open`)
-			}
+			tracing.LogKV("circuit_breaker", "open", ctx.request.Context())
```

Comment:
I added it to the ingress span in case proxy.do() returns an error, because the client does not care which part errors and the proxy span already sets this tag on error in proxy.makeBackendRequest